ballerina/tests/secure_client_endpoint_test.bal

```diff
@@ -82,15 +82,15 @@ public function testSecureGetFileContentWithWrongPassword() returns error? {
             if (arr1 is error) {
                 log:printError("Error while `next` operation " + arr1.message());
             } else {
-                test:assertTrue(true);
+                test:assertFail(msg = "Found unexpected response type");
             }
         }
         io:Error? closeResult = str.close();
         if (closeResult is io:Error) {
             test:assertFail(msg = "Error while closing stream in `get` operation.");
         }
     } else {
-       test:assertFail(msg = "Found unexpected response type" + str.message());
+       test:assertTrue(true);
```

Comment:
`test:assertTrue(true);` can not we remove these obvious results in some of the if-else condition branches?

Reply:
Replaced with error message validation.